lib/import/BpmnTreeWalker.js

```diff
@@ -83,7 +80,7 @@ export default function BpmnTreeWalker(handler, translate) {
     // avoid multiple rendering of elements
     if (gfx) {
       throw new Error(
-        translate('already rendered {element}', { element: elementToString(element) })
+        'already rendered ${ elementToString(element) }'
```

Comment:
tagged template is missing, has been modified in this [pr ](https://github.com/bpmn-io/bpmn-js/pull/2151)